runtime/doc/autocmd.txt

```diff
@@ -812,6 +812,14 @@ RemoteReply			When a reply from a Vim that functions as
 				Note that even if an autocommand is defined,
 				the reply should be read with |remote_read()|
 				to consume it.
+							*RecordingStart*
+RecordingStart			When a macro starts to be recorded.
```

Comment:
no precedent for "Start"/"Stop" in event names. I guess "Enter"/"Leave" make sense?

Reply:
That's what I thought too, but it felt a bit weird... But if that's what you want, I'll do t

Reply:
I'm ok with either.

Reply:
Went for `Enter` and `Leave` for consistence.